src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/MenuItem.cs

```diff
@@ -2498,7 +2498,10 @@ private void FocusOrSelect()
             // but if we fail to focus we should still select.
             // (This is to help enable focusless menus).
             // Check IsKeyboardFocusWithin to allow rich content within the menuitem.
-            if (!IsKeyboardFocusWithin)
+            if (!IsKeyboardFocusWithin
+                // But only aquire focus if the window we are inside of is currently active or there is no window.
```

Comment:
Nit:
```suggestion
                // But only acquire focus if the window we are inside of is currently active or there is no window.
```

Suggested rewording (Makes it more clear to me but feel free to dismiss this suggestion):
```suggestion
                // But only acquire focus if the window containing this menu item is currently active or if this menu item is not in a window.
```

Reply:
Will fix the typo, but the comment above mine also talks about "we".
I don't think it helps if we write out in which class we currently are as it's a source comment, that seems redundant.

Reply:
```
I don't think it helps if we write out in which class we currently are as it's a source comment, that seems redundant.
```
Fair enough.

Reply:
This conversation seems resolved.